postgres_backend.go

```diff
@@ -82,22 +83,28 @@ type PgBackend struct {
 // - WithTransactionTimeout(timeout int): configure the idle_in_transaction_timeout for the worker's database
 // connection(s)
 func NewPgBackend(ctx context.Context, connectString string, opts ...ConfigOption) (n Neoq, err error) {
-	w := PgBackend{
-		mu:         &sync.Mutex{},
-		config:     &pgConfig{connectString: connectString},
-		handlers:   make(map[string]Handler),
-		futureJobs: make(map[int64]time.Time),
-		logger:     slog.New(slog.NewTextHandler(os.Stdout)),
-		cron:       cron.New(),
+	w := &PgBackend{
+		mu:          &sync.Mutex{},
+		config:      &pgConfig{connectString: connectString},
+		handlers:    make(map[string]Handler),
+		futureJobs:  make(map[int64]time.Time),
+		logger:      slog.New(slog.NewTextHandler(os.Stdout)),
```

Comment:
Log should be passed in. Maybe a WithSLog option

Reply:
There's a TODO in the code for that already. I'm going to defer on this for now.